docs/python/machine_learning/synthethic_data/1_synthetic_data.md

```diff
@@ -0,0 +1,178 @@
+# Introduction
```

Comment:
Lets add a subsection for Personal Identifiable Information, just to inform what it is